src/Core/src/Platform/iOS/ButtonExtensions.cs

```diff
@@ -83,5 +84,24 @@ public static void UpdatePadding(this UIButton platformButton, Thickness padding
 #pragma warning restore CA1422 // Validate platform compatibility
 #pragma warning restore CA1416
 		}
+
+		internal static void UpdateAttributedTitle(this UIButton platformButton, IFontManager fontManager, ITextStyle textStyle)
+		{
+			// Any text update requires that we update any attributed string formatting
+			var uiFontAttribute = fontManager.GetFont(textStyle.Font, UIFont.ButtonFontSize);
+			var attributedString = new NSMutableAttributedString(new NSAttributedString(platformButton.CurrentTitle!));
+
+			// Update font family & size
+			attributedString.AddAttribute(UIStringAttributeKey.Font, uiFontAttribute, new NSRange(0, attributedString.Length));
+
+			//Update UpdateCharacterSpacing
+			attributedString = attributedString.WithCharacterSpacing(textStyle.CharacterSpacing);
```

Comment:
Although my little testing code sample passed, it looks like if the character spacing is set to 0, the WithCharacterSpacing method will reset the attributedString to null.

```suggestion
			//Update UpdateCharacterSpacing
			if (textStyle.CharacterSpacing != 0)
			{
			attributedString = attributedString.WithCharacterSpacing(textStyle.CharacterSpacing);
			}
```